Packs/SortListOfDictsByKey/Scripts/SortListOfDictsByKey/SortListOfDictsByKey.py

```diff
@@ -0,0 +1,47 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+"""
+Sort a list of dictionaries by any key of the dictionaries. Can sort in reverse if set to true.
+"""
+
+
+def sort_list_of_dicts_by_key(args: Dict[str, Any]) -> CommandResults:
+
+    _list = args.get('value', None)
+    key = args.get('key', None)
+    reverse_flag = args.get('reverse', False)
+
+    if reverse_flag == 'True':
+        reverse = True
+    else:
+        reverse = False
+
+    if not _list:
+        raise ValueError('List not provided')
+
+    result = _list.sort(key=lambda x: x[key], reverse=reverse)
```

Comment:
This result is never returned to the user, make sure to avoid unused parameters

Reply:
Removed `result` from source